flow/record/fieldtypes/__init__.py

```diff
@@ -545,8 +545,10 @@ def __new__(cls, *args):
                 path_ = args[0]
                 if isinstance(path_, pathlib.PureWindowsPath):
                     cls = windows_path
+                    args = [path_.as_posix()]
```

Comment:
Interesting issue. This feels like a problem with `pathlib.PureWindowsPath` if initializing it with a `PureWindowsPath` constructs it as a slightly different `PureWindowsPath`.

Reply:
Yes, think it's also an issue in Python.